### PR TITLE
Fix/wallet small fixes

### PR DESCRIPTION
**Pull request by Cuteivist**
### What does the PR do

fixes: #10374
fixes: #10394 

Notes

* review commit by commit

### Affected areas

Wallet screen back button
Wallet Saved Addresses view

https://user-images.githubusercontent.com/11396062/234346986-8c6974e3-5882-4ccc-b521-dc8fe258fe53.mp4

### CODE REVIEW

**Comment by caybro** [comment]
@Cuteivist pls rebase to get rid of the git conflicts

**Review by stefandunca — APPROVED** [review]
Good job @Cuteivist!

**Comment by Khushboo-dev-cpp** [comment]
@Cuteivist Great job :)


https://user-images.githubusercontent.com/60327365/234508319-cf1fea68-aacc-4755-abbc-6bccedd8d5bd.mov

Only thing is when we switch between saved address and wallet accounts we see a small flicker at the WalletFooter and I wonder if saved addresses can be moved inside the Right Side view, or the footer be made permanent somehow to avoid that flicker?

**Comment by Cuteivist** [comment]
@caybro Fixed conficts
@Khushboo-dev-cpp I fixed that issue by moving `WalletFooter` from components to Wallet Layout view.

Please see updated video:

https://user-images.githubusercontent.com/11396062/234521603-db1bbc48-240a-40dd-be83-b40bf73152c7.mp4

**Comment by Khushboo-dev-cpp** [comment]
@Cuteivist  maybe its better if we add a footer here for anyone who needs it ?

See this , they added a header here
https://github.com/status-im/status-desktop/blob/663fac9fa76fcdb008d58f8548c88790ad4d2c3b/ui/StatusQ/src/StatusQ/Layout/StatusSectionLayout.qml#L155

**Comment by Cuteivist** [comment]
@Khushboo-dev-cpp I don't know the specs for whole application, but do we need WalletFooter in every view where `StatusSectionLayout` is used? We could put it inside the Loader so we will load on request, e.g. in WalletLayout but on other hand it creates problem that StatusSectionLayout will break single responsibility rule.

In my opinion if WalletFooter is used only in WalletLayout then my solution is more correct. If it's planned to be shown in nearly all views (e.g. chat, profile) then placing it in StatusSectionLayout makes sense.

**Comment by Khushboo-dev-cpp** [comment]
> @Khushboo-dev-cpp I don't know the specs for whole application, but do we need WalletFooter in every view where `StatusSectionLayout` is used? We could put it inside the Loader so we will load on request, e.g. in WalletLayout but on other hand it creates problem that StatusSectionLayout will break single responsibility rule.
> 
> In my opinion if WalletFooter is used only in WalletLayout then my solution is more correct. If it's planned to be shown in nearly all views (e.g. chat, profile) then placing it in StatusSectionLayout makes sense.

Hey @Cuteivist , I do not mean movin the WalletHeader into the StatusSectionLayout but simple creating a slot for it, likes is done for header, so that it can be used if needed and if not, it doesnt affect anyone.

File: ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml [hunk 1]
```diff
@@ -99,15 +100,14 @@ Item {
...
-        objectName: "SavedAddressesView_savedAddresses"
-        anchors.top: errorMessage.bottom
+        anchors.top: header.bottom
         anchors.topMargin: Style.current.padding
         anchors.bottom: parent.bottom
-        anchors.bottomMargin: Style.current.halfPadding
         anchors.right: parent.right
         anchors.left: parent.left
-        visible: listView.count > 0
+        objectName: "SavedAddressesView_savedAddresses"
```
[the remark this answers is not in the document]
Comment (Cuteivist) [reply ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml]: Fixed. Good catch!

**Review by Khushboo-dev-cpp — APPROVED** [review]
LGTM !! Awesome job:)

**Pull request merged**